libraries/chain/controller.cpp

```diff
@@ -3459,7 +3459,7 @@ struct controller_impl {
             fc::time_point now = fc::time_point::now();
 
             ilog("Produced block ${id}... #${n} @ ${t} signed by ${p} "
-                 "[trxs: ${count}, lib: ${lib}${confs}, net: ${net}, cpu: ${cpu}, elapsed: ${et} us, time: ${tt} us]",
+                 "[trxs: ${count}, lib: ${lib}${confs}, net: ${net}, cpu: ${cpu} us, elapsed: ${et} us, producing time: ${tt} us]",
```

Comment:
I'm not sure how a user would understand the difference between `elapsed` and `producing time`?